db/migrate/20190208051452_add_jwt_hash_to_exchange_rate.rb

```diff
@@ -0,0 +1,5 @@
+class AddJwtHashToExchangeRate < ActiveRecord::Migration[5.2]
+  def change
+    add_column :bloom_trade_client_exchange_rates, :jwt_hash, :string
```

Comment:
To avoid sequential scans, this should be a composite index (with the buy and sell currencies)